### ISSUE TITLE
When Backdrop Clicked doesn't register when clicking on a blank section of a backdrop

### ISSUE THREAD

**Issue by thisandagain** [LLK/scratch-gui#1668]
Expected Behavior
Clicking the stage somewhere there is no art should result in the When Stage Clicked block activating.

Actual Behavior
If a backdrop of the sprite has an area that has no art in it and you click that part on the stage the When Stage Clicked block doesn't activate.

Note: This is not the case when the default costume is used for the stage, but is the case if you paint a new backdrop or if you modify the default backdrop.

Repro Steps
Open a new Scratch 3.0 project
Select the stage
Click the costumes tab
Draw a shape in the editor.
Click back to blocks
Build a stack that is When Stage Clicked > Play Sound Pop
Click the stage where it is white
Note: the lack of a pop sound
Click the stage in a part that you have added a shape
Note: A pop!

Paul asked me to bring mention his to @cwillisf 

---
Moved from LLK/scratch-render#249
/cc @BryceLTaylor @cwillisf

**Comment by mrjacobbloom** [comment]
In 2.0, it's not possible to create a see-through backdrop. If you flood transparency onto the stage, it's replaced by white when you navigate away. Even when you're still in the paint editor and can see that the stage is transparent, the "when Stage clicked" block still runs when the stage is clicked

**Comment by Mr-Dave2** [comment]
I saw this report after submitting a “feedback” report on what I think is the same issue.

My project has a “wrong stage clicking” behavior if a stamp is placed on the background. I think it can be made to match the problem in this post by creating an empty stage background.  Doing that after the project has started will show another problem in that the “say” bubbles will change.  One will disappear and the other will be moved lower.

Link to my test project:
https://preview.scratch.mit.edu/#214745715

Scratch 2 link at https://scratch.mit.edu/projects/214745715/

Scratch 3.0 acts as if the stage area is just the “costume” of the current backdrop as opposed to the area of the stage where there are no sprites.

**Comment by paulkaplan** [comment]
Yeah looks like that is a related part of this bug, the stage should register a click if there are no sprites there, including if the click is on a pen stamp. https://github.com/LLK/scratch-gui/issues/1840 is another example of this.
